bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -112,6 +120,15 @@ private synchronized void poll() {
         }
     }
 
+    private void stopPoll() {
+        final Future<?> future = pollFuture;
+        if (future != null && !future.isCancelled()) {
+            future.cancel(true);
+            pollFuture.cancel(true);
```

Comment:
This one can be removed, the future is already cancelled above.

Reply:
Fixed in cda422cf1c.